mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -58,13 +93,13 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
       require(sample.size > 0, s"Vector should have dimension larger than zero.")
       n = sample.size
 
```

Comment:
having
```scala
val withMean = configure("mean")
val withVariance = configure("variance")

if (withMean || withVariance)
...
```

Since it's `val`,  JIT can kick in for optimization.